### PR TITLE
feat: add cursor pointer in PbRadio

### PR DESCRIPTION
**Pull request by alanneves**
in this PR i'm adding cursor pointer in the radio input component

### CODE REVIEW

**Review by mickaelnn — APPROVED** [review]
LGTM

**Review by caroltrindade — APPROVED** [review]
LGTM

File: src/components/Inputs/Radio/Radio.vue [hunk 1]
```diff
@@ -20,10 +12,10 @@
 
 <script>
 export default {
-  name: 'PbRadio',
+  name: "PbRadio",
```
Comment (Viserion77) [inline src/components/Inputs/Radio/Radio.vue:15]: voltar para aspas simples

**Review by Viserion77 — CHANGES_REQUESTED** [review]

**Review by caroltrindade — APPROVED** [review]
LGTM

**Review by Viserion77 — APPROVED** [review]
LGTM

**Review by mickaelnn — APPROVED** [review]
LGTM

**Pull request merged**